### DISCUSSION TITLE
different sunrise results using two different examples of the library

### DISCUSSION THREAD

**Discussion by getoledo** [jpb10/SolarCalculator#14] [Q&A] [answered]
Hello, Nice job !
I am testing the library, and there is something I don´t understand
If I use the example "SunriseSunset" with latitude = 40.457497 , longitude = -3.454602, utc_offset = 1 , the sunrise outputs is 08:19 hs.
Then, if now I use example "SolarTrackingTimeLib" with same coordinates, and // Set time manually (hr, min, sec, day, mo, yr)
setTime(8, 19, 0, 2, 12, 2022) (same sunrise hour as in the previous example), this result in Elevation of :8.84º instead of 0º according to sunrise.
There is an error of 1 hour in some place, because if I use setTime(7 19, 0, 2, 12, 2022) (7:19hs) this gives me an Elevation of -0.33 (sunrise)
Can you help me to figure it out ?

By the way, in same example if I change utc_offset , nothing happens... what should happen?

Thanks !

**Comment by jpb10** [comment, marked as answer]
All input and output times are in UTC (i.e. GMT). Local time 08:19 CET = 07:19 GMT. 

That's why `setTime(7, 19, 0, 2, 12, 2022)` gives the correct elevation.

**Reply by getoledo** [reply]
Thank you my friend !
